code/espurna/sensor.cpp

```diff
@@ -2034,9 +2034,17 @@ void _sensorLoad() {
 
     #if VL53L1X_SUPPORT
     {
+        #if (VL53L1X_INTER_MEASUREMENT_PERIOD <= (VL53L1X_MEASUREMENT_TIMING_BUDGET + 4))
+            #error "[VL53L1X] Intermeasurement period must be greater than the timing budget + 4ms"
+        #endif
+
+        #if (VL53L1X_MEASUREMENT_TIMING_BUDGET < 20 || VL53L1X_MEASUREMENT_TIMING_BUDGET > 500)
+            #error "[VL53L1X] Timing budget is limited to [20, 500]"
+        #endif
```

Comment:
idk if this is necessary here

- looking at the library source, you could just check setMeasurementTimingBudget() boolean result by proxying it via the sensor method and setting internal error
- this could also happen with setInterMeasurementPeriod(), never allowing startContiniuous in the begin()

thus, replacing `#error` with debug_msg_p() call